lib/src/value/calculation.dart

```diff
@@ -341,15 +342,23 @@ class CalculationOperation {
   ///
   /// This is either a [SassNumber], a [SassCalculation], an unquoted
   /// [SassString], a [CalculationOperation], or a [CalculationInterpolation].
-  final Object left;
+  Object get left {
+    return _left;
+  }
+
+  final Object _left;
```

Comment:
Style nit: use expression-bodied methods when possible, especially when wrapping fields.

```suggestion
  Object get left => _left;
  final Object _left;
```